app/controllers/v0/facilities/ccp_controller.rb

```diff
@@ -55,16 +52,20 @@ def api
     @api ||= Facilities::PPMS::Client.new
   end
 
+  def per_page
+    Integer(search_params[:per_page] || BaseFacility.per_page)
+  end
+
   def page
-    Integer(params[:page] || 1)
+    Integer(search_params[:page] || 1)
   end
 
-  def pages(ppms_results)
-    total = ppms_results.length
+  def pages(_ppms_results)
+    total = (page + 1) * per_page
     {
       current_page: page,
-      per_page: BaseFacility.per_page,
-      total_pages: Integer(ppms_results.length / BaseFacility.per_page + 1),
+      per_page: per_page,
+      total_pages: page + 1,
```

Comment:
With our conversation this in mind, does it make sense for this value be `page` when `ppms_results.empty?`
Is it possible in that scenario that they go to the next page and get results?